code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -1,26 +1,115 @@
 ---
 openapi: 3.0.3
 info:
-  title: Edge Cloud API
-  version: 0.9.2
-  description:  |
-    Edge Cloud API for Life Cycle Management and discovery of an application.
+  title: Edge Application Management API
+  version: 0.9.3-wip
+  description: |
+    Edge Application Management API allows API consumers to manage the
+    Life Cycle of an Application and to Discover Edge Cloud Zones.
+    # Overview
+    The reference scenario foresees a distributed Telco Edge Cloud where any
+    Application Delevoper, known as an Application Provider, can host and
+    deploy their application according to their specifications and operational
+    criteria (e.g. within an specific geographical zone for data protection
+    purposes, ensure a minimum QoS for the application clients, etc).
+    Through Telco Edge Cloud services Developers around the globe can be
+    benefit from the traditional Cloud strengths but expertise and advantages
+    of the Mobile Network Operators offering to their users an evolved
+    experience for XR, V2X, Holographic and other new services.
+
     # Introduction
-    APIs defined in this version of the specification can be categorized into
-    the following areas:
-    * __Application__ - The Application Provider submit application metadata
-    to the Operator Platform. The OP generate an appId for that metadata that
-    will be used to instantiate the application within the Edge Cloud Zone.
-    * __Edge Cloud information__ - Retrieves all the Edge Cloud Zones
-    available according to some defined parameters where an application can
-    be instantiated.
-    # Relevant terms and definitions
-    This section provides definitions of terminologies commonly referred
+    The Edge Application Management API provides capabilities for lifecycle
+    management of application, instances and edge discovery.
```

Comment:
For consistency do we need to change "edge discovery" to "edge cloud zone" discovery"